### DISCUSSION TITLE
어떤 방식으로 해야 하나요?

### DISCUSSION THREAD

**Discussion by Seungjun07** [KimMin-Gwan/ICT-convergence-project-contest-2023#28] [General]
현재 ir센서 조작을 파이썬 코드로 구현하였는데 시간에 따라 동작하도록 구현하였습니다. 
다른 좋은 방법이 있다면 조언 부탁 드려요.

**Comment by KimMin-Gwan** [comment]
혹시 리모컨의 IR값을 받아오는 방식은 어떻게 되나요?
cmd창에 나온 값을 하드코딩하여 입력하는 방식입니까?
아니면 자동으로 입력된 값을 사용합니까?

동적으로 어떤 리모컨이든 쉽게 등록하고 사용하게 하려고 합니다.
좋은 방법이 있거나 유추 할만한 단서가 있다면 알려주세요.

**Comment by KimMin-Gwan** [comment]
금요일부터 커밋하신 코드를 바탕으로 라즈베리파이에 넣을 프로그램을 제작할 것입니다.
